http_handlers.go

```diff
@@ -124,10 +150,54 @@ func getStringFromIndex(data []byte, index int) string {
 	return string(data[start:end])
 }
 
+//make an index on column values in dataset
+func makeIndex() {
+
+	sort.Slice(ITEMS, func(i, j int) bool {
+		return ITEMS[i].Buurtcode < ITEMS[j].Buurtcode
+	})
+
+	LOOKUP = make(map[string]Items)
+	kSet := make(map[string]bool)
+
+	for _, item := range ITEMS {
+		key := strings.ToLower(item.Buurtcode)
+		kSet[key] = true
+		LOOKUP[key] = append(LOOKUP[key], item)
+	}
```

Comment:
Let's create a "getIndex" that is created in the models.go file.
Then this code doesn't have to use "Buurtcode but instead will call getIndex.
```
sort.Slice(ITEMS, func(i, j int) bool {
		return ITEMS[i].getIndex() < ITEMS[j].getIndex()
	})
```

```
key := strings.ToLower(item.getIndex())
```
models.go
```
func (i item*) getIndex() string {
     return i.Buurtcode
}
```

Reply:
fixed.